indexer/packages/postgres/src/models/liquidity-tiers-model.ts

```diff
@@ -27,6 +27,9 @@ export default class LiquidityTiersModel extends BaseModel {
         name: { type: 'string' },
         initialMarginPpm: { type: 'string', pattern: IntegerPattern },
         maintenanceFractionPpm: { type: 'string', pattern: IntegerPattern },
+        // optional fields which are nullable
```

Comment:
nit: you don't need to include the comment, this is expected since they're not in the required fields and this behavior is shown through the `postgres/src/models` directory in multiple places